### PR TITLE
Switch to MIT License

### LINKED ISSUE #28: Bug in LICENSE.txt
There seems to be a bug in LICENSE.txt that stems from Section 3a (https://github.com/ni/VireoSDK/blob/incoming/LICENSE.txt#L18-L19) that prevents us from running Vireo.

This issue might also affect young learners seeking jobs in our profession.

This issue manifests on all platforms.

Perhaps boilerplate `MIT license` could resolve this issue?

Promising platform, Vireo!

### PR DESCRIPTION
**Pull request by rajsite**
It was [approved](https://niweb.natinst.com/confluence/display/LV/Make+Vireo+--+the+WebVI+Runtime+Engine+--+Open+Source+under+an+MIT+License) to update the license for [VireoSDK](https://github.com/ni/VireoSDK) to the MIT License.

With [some (mostly in person) discussion](https://github.com/orgs/ni/teams/ni/discussions/9) with the open source guild the following was deemed reasonable to enable the transition in this PR:

- Replace the contents of [License.txt](https://github.com/ni/VireoSDK/blob/master/LICENSE.txt) with the [MIT License](https://opensource.org/licenses/MIT) text
- Update the [npm package](https://github.com/ni/VireoSDK/blob/ccb691e9ce9f62ed40375dceedd4f14ef99cc0d3/package.json#L52) and nuget package license fields as needed
- Update license references in README and docs (as opposed to the license of the doc themselves)
- Add / update the license header in all `source` files:
   - Source code (*.cpp *.h *.js *.via *.java *.g4 *.php)
   - Infrastructure files (Makefiles *.yml *.sh *.gyp *.bat *.xml)
   - Docs (*.md) - The license for the documentation itself
 
And we can use an [SPDX ID](https://spdx.org/ids) for the header. The following is the header with a [short identifier](https://spdx.org/sites/cpstandard/files/pages/files/using_spdx_license_list_short_identifiers.pdf):

```js
// Copyright (c) 2020 National Instruments
// SPDX-License-Identifier: MIT
```
In addition, a note about SPDX usage was added in the license file as discussed on [page 14 of _Using SPDX License List “short identifiers” in source files_](https://spdx.org/sites/cpstandard/files/pages/files/using_spdx_license_list_short_identifiers.pdf).

I'll *not* be adding the header to "data files" (*.json *.vtr), IDE config files (for XCode, Visual Studio), or via files that are part of the "vtr test suite" located in /test-it/ViaTests (as there are tests for parsing via itself where line number reporting is significant). The open source guild said this was fine, primarily wanting to ensure the root License file update.

Fixes #28

### CODE REVIEW

File: .appveyor.yml [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# Copyright (c) 2020 National Instruments
+# SPDX-License-Identifier: MIT
```
Comment (sanmut) [inline .appveyor.yml]: Very nice to include the SPDX header 🥇

**Comment by sanmut** [comment]
Thanks for all the hard-work fixing issue #28  !!!

File: .appveyor.yml (hunk 1 above)
Comment (rajsite) [reply .appveyor.yml]: The brevity and ability to automate detection are definitely nice :)

**Comment by rajsite** [comment]
Note: Do not merge yet, pending a discussion on adding a [THIRDPARTY.txt](https://github.com/rajsite/VireoSDK/commit/05dc12aba1f3326dff0b22ddf4f57d124a65acbf) file.

**Comment by rajsite** [comment]
> Note: Do not merge yet, pending a discussion on adding a [THIRDPARTY.txt](https://github.com/rajsite/VireoSDK/commit/05dc12aba1f3326dff0b22ddf4f57d124a65acbf) file.

Got feedback that adding a THIRDPARTY.txt for use in distributions works, can merge when ready

**Pull request merged**